overlord/devicestate/handlers_install.go

```diff
@@ -141,10 +142,7 @@ func (m *DeviceManager) doSetupRunSystem(t *state.Task, _ *tomb.Tomb) error {
 	return nil
 }
 
-// TODO:UC20: set to real TPM availability check function
-var checkTPMAvailability = func() error {
-	return nil
-}
+var checkKeySealingSupported = secboot.CheckKeySealingSupported
```

Comment:
traditionally we would have called this:  secbootCheckKeySealingSupported  not a blocker